### ISSUE TITLE
resource is denied error

### ISSUE THREAD

**Issue by robertmehrer** [adilinden-oss/docker-rpi-stream#1]
when running docker build -t adilinden/rpi-stream . i get this error

Sending build context to Docker daemon  183.8kB
Step 1/4 : FROM adilinden/rpi-ffmpeg:latest
pull access denied for adilinden/rpi-ffmpeg, repository does not exist or may require 'docker login': denied: requested access to the resource is denied

**Comment by Syndael** [comment]
> 
> 
> when running docker build -t adilinden/rpi-stream . i get this error
> 
> Sending build context to Docker daemon 183.8kB
> Step 1/4 : FROM adilinden/rpi-ffmpeg:latest
> pull access denied for adilinden/rpi-ffmpeg, repository does not exist or may require 'docker login': denied: requested access to the resource is denied

+1

**Comment by sonjamichelle** [comment]
Same error here. Is the git still under development?

**Comment by kolisko** [comment]
@Syndael  @sonjamichelle I created the PR adilinden-oss/docker-ffmpeg#1 fix and built the kolisko/rpi-stream image

**Comment by sonjamichelle** [comment]
@kolisko So we can follow the original instructions now and everything will work?

**Comment by kolisko** [comment]
Yes, follow the original instructions and replace the original image by the new one
